src/Tribe/Views/V2/View.php

```diff
@@ -65,6 +65,17 @@ class View implements View_Interface {
 	 */
 	protected $slug = 'view';
```

Comment:
Since this is deprecated in classes extending this one, I would update this class `__construct` method to set the instance `slug` property to the value of the static `view_slug` property to make sure current users of the instance `slug` property` will get the correct value.